iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/KubernetesVerifier.java

```diff
@@ -297,6 +297,7 @@ public static class KubernetesTestContext extends Verifier.TestContext implement
     private final InputFileContext inputFileContext;
     private final ProjectContext projectContext;
     private boolean shouldReportSecondaryInValues = true;
+    private boolean disableLocationShifting;
```

Comment:
I'd make the opposite default: `enableLocationShifting = true`, because we use this for the most rules, and it can be more readable like this

Reply:
Was my first idea, wasn't sure though. I will change it